### PR TITLE
Refactoring for readability

### PR DESCRIPTION
**Pull request by cooksey**
* Moves Zello-related code into Zello object
* Separates driver/rider screen into DriverRiderFragment

### CODE REVIEW

File: sdks/android/demo/sample-ride/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
             android:name=".MainActivity"
             android:configChanges="keyboard|keyboardHidden|screenLayout|orientation|screenSize"
             android:label="@string/app_name"
-            android:screenOrientation="sensor"
+            android:screenOrientation="portrait"
```
Comment (zello-eugene) [inline sdks/android/demo/sample-ride/src/main/AndroidManifest.xml]: I may be incorrect on this one, but I think in the past we had a design from @WilNichols that included a landscape-specific layout, too. Any particular reason to lock the orientation at this time?

Comment (cooksey) [reply sdks/android/demo/sample-ride/src/main/AndroidManifest.xml]: It looked like it was going to be a lot of work for me to understand and maintain, and it didn't help at all for the Channels SDK demonstration purposes of this app.

File: sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+
+	override fun onPttButtonUp(cancelled: Boolean) {
+		Zello.instance.outgoingStream?.stop()
+	}
+
+	// endregion
+
+	// Called to set up the screen when the driver mode is activated
+	fun setupDriver() {
+//		if (session == null) {
```
Comment (zello-eugene) [inline sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt]: Looks like this was moved to the `Zello` singleton (I appreciate the refactoring). Can  the commented code here and below be removed?

File: sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+		listener = (context as? MainActivity)?.driverRiderListener
+	}
+
+	override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+		if (inflater == null) {
+			return super.onCreateView(inflater, container, savedInstanceState)
+		}
+		this.savedInstanceState = savedInstanceState
+
+		// TODO: Inflate correct layout
```
Comment (zello-eugene) [inline sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt]: Quite a number of TODOs between lines 68 and 90. Are these meant to be looked at during this pass?

File: sdks/android/demo/sample-ride/src/main/AndroidManifest.xml (hunk 1 above)
Comment (zello-eugene) [reply sdks/android/demo/sample-ride/src/main/AndroidManifest.xml]: That makes sense. I just thought it worked in the past.

File: sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt (hunk 3 above)
Comment (cooksey) [reply sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt]: They're already done; I forgot to remove the TODOs.

File: sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt (hunk 2 above)
Comment (cooksey) [reply sdks/android/demo/sample-ride/src/main/java/com/zello/sample/ride/DriverRiderFragment.kt]: Yup, done.

**Pull request merged**